blint/lib/disassembler.py

```diff
@@ -52,7 +52,7 @@ def _get_function_ranges(parsed_obj, metadata):
                      for sec in parsed_obj.sections:
                          sec_start = sec.virtual_address + parsed_obj.optional_header.imagebase
                          if sec_start <= func_addr < sec_start + sec.virtual_size:
-                             sec_identifier = sec.virtual_address
+                             sec_identifier = sec.virtual_address + parsed_obj.optional_header.imagebase
```

Comment:
One day, Blint will have a contributor who can review and help me hunt these sorts of bugs.